cpu/native/Makefile.dep

```diff
@@ -40,9 +40,16 @@ ifneq (,$(filter socket_zep,$(USEMODULE)))
   endif
 endif
 
-ifneq (,$(filter ztimer_xtimer_compat,$(USEMODULE)))
-  # requires 64bit for syscalls
-  USEMODULE += ztimer64_xtimer_compat
+ifneq (,$(filter libc_syscalls_gettimeofday,$(USEMODULE)))
+  USEMODULE += xtimer
+  ifneq (,$(filter ztimer_xtimer_compat,$(USEMODULE)))
+    # requires 64bit timestamps
+    USEMODULE += ztimer64_xtimer_compat
+  endif
+endif
+
+ifneq (,$(filter tinydtls lua wolfssl,$(USEPKG)))
+  USEMODULE += libc_syscalls_gettimeofday
```

Comment:
shouldn't the these be checked before `libc_syscalls_gettimeofday` is checked? (so above l.:43)

Reply:
Hmm actually the dependency is in the pkg, I think this can be removed.

Reply:
I'll move it to the pkg, and check for native.